keyboards/giabalanai/info.json

```diff
@@ -25,12 +150,11 @@
         "midi": true,
         "extrakey": true,
         "encoder": true,
+        "encoder_map": true,
```

Comment:
`encoder_map` is never allowed to be enabled at the keyboard level.

All `rules.mk` changes regarding `encoder_map` will also need to be reverted.
```suggestion
```